plugins/mcreator-localization/help/default/item/immune_to_fire.md

```diff
@@ -1,3 +1,4 @@
 This parameter controls if the item is immune to fire, like Netherite stuff.
 
-NOTE: Only available for Minecraft 1.16.4+
\ No newline at end of file
+
+NOTE: Only available in Minecraft 1.16.x and higher
```

Comment:
You changed that for english, @Goldorion should change it for French too

Reply:
Klemen changed it, but I will change it.

Reply:
I think Max meant that I changed it and that you should change them for FR too.

Reply:
I understood that way too. I meant will change it in French tips.